backend/src/testsetup/setupTests.ts

```diff
@@ -0,0 +1,40 @@
+import { database } from "../config/database";
+import { MuralCollection } from "../models/muralcollection.model";
+import { Tour } from "../models/tour.model";
+import { Mural } from "../models/mural.model";
+import { Borough } from "../models/borough.model";
+import { Artist } from "../models/artist.model";
+
+beforeAll(async () => {
```

Comment:
did you replace the individual `beforeAll`s with this one?

Reply:
correct, the beforeAll/afterAll in this file run before and after every individual test suite. We had a beforeAll setting up the sequelize associations and an afterAll disconnecting from the database in basically every test suite, so it is way cleaner to put them all here.